examples/create-react-app-with-styled-components/README.md

```diff
@@ -16,9 +16,14 @@ npm install
 npm start
 ```
 
-or:
+## CodeSandbox
+
+Note that CodeSandbox is not supporting react-app-rewired, yet you can [still see the code](https://codesandbox.io/s/github/mui-org/material-ui/tree/HEAD/examples/create-react-app-with-styled-components).
+
+The following link leverages this demo: https://next.material-ui.com/guides/interoperability/#change-the-default-styled-engine with Parcel's alias feature within the `package.json`
+
+[![Edit on CodeSandbox](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/styled-components-forked-guoes)
```

Comment:
We can only link sandboxes Material-UI host under it's organization. To minimize duplicated work, let's use the existing one 

```suggestion
[![Edit on CodeSandbox](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/styled-components-interoperability-w9z9d)
```

Reply:
Ok, then if you can please change 
```js
import { experimentalStyled as styled } from "@material-ui/core/styles";
```

to :

```js
import styles from "styled-components";
```

in `demo.js` so it will perfectly fit the exmaple

Reply:
Our custom styled helper because it comes with extra features compared to the raw styled helper. What's the advantage of importing from `styled-components` directly?

Reply:
Since this code representing the code that one will write in his own app. so I we should be as close as possible to external code that actually customizing the component with `styled` from `styled-components`

Reply:
Ok, I think that we should keep the current display:

1. It will bring awareness that this option exists. If developers want to use the import raw, they should already know how to do it.
2. We will encourage using the wrapper, for the extra features that come with it.

Reply:
ok!